server/routes/api/helpscout.ts

```diff
@@ -0,0 +1,126 @@
+import * as express from "express";
+import {isHelpScoutIntegrationEnabled, validateHelpScoutSignature} from "../../middleware";
+import {IFormItem, IUser, User} from "../../schema";
+import bodyParser = require("body-parser");
+import * as moment from "moment-timezone";
+
+export const helpScoutRoutes = express.Router({"mergeParams": true});
+
+export type RequestWithRawBody = express.Request & { rawBody: string };
+
+helpScoutRoutes.route("/userInfo").post(
+	isHelpScoutIntegrationEnabled,
+	bodyParser.json({
+		verify: (req: RequestWithRawBody, res, buffer, encoding) => {
+			if (buffer && buffer.length) {
+				req.rawBody = buffer.toString(encoding || 'utf-8');
+			}
+		}
+	}),
+	validateHelpScoutSignature,
+	helpScoutUserInfoHandler
+);
+
+async function findUserByEmail(email: string) {
+	return User.findOne({
+		email
+	});
+}
+
+function findApplicationQuestion(applicationData: IFormItem[], questionName: string) {
+	const result = applicationData.find((item: IFormItem) => item.name === questionName);
+	return result ? result.value : "";
+}
+
+function createListBlock(title: string, content: string | string[] | Express.Multer.File | null) {
+	return `<li class="c-sb-list-item">
+			<span class="c-sb-list-item__label">${title}<span class="c-sb-list-item__text">${content}</span>
+		</span>
+	</li>`;
+}
+
+function badge(type: string, text: string) {
+	return `<span class="badge ${type}">${text}</span>`;
+}
+
+async function helpScoutUserInfoHandler(request: express.Request, response: express.Response) {
+	const email = request.body.customer.email;
+	const user: IUser | null = await findUserByEmail(email);
+
+	if (!user) {
+		response.status(200).json({
+			html: `<h4><a href="mailto:${email}">${email}</a></h4>
+				<h4>There are no users in registration with this email</h4>`.replace(/[\n\t]/g, "")
+		});
+	} else {
+		let affiliationName: string;
+		if (user.applicationBranch === "Mentor") {
```

Comment:
Generally we've tried to keep specific strings like this contained to event branches, so if you backport this to master maybe think of some way to make the display more generic/parameterized. Maybe add an optional field for questions to determine if they show up in helpscout? Like `basicInfo: true`

Reply:
Was thinking about this too but via environment variables but I think this is a better way to handle it.  Might have to think more about the exact way to do this though

Reply:
I love this suggestion, and implemented it with a new `showInHelpScout` question schema option